docker/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM ubuntu:18.04
+MAINTAINER patrick.sharkey@nextcentury.com
+
+# Prevent dialog during apt install
+ENV DEBIAN_FRONTEND noninteractive
+
+# Update system packages and remove any downloaded files
+RUN \
+  rm -rf /var/lib/apt/lists/* \
+  && apt-get clean \
+  && apt-get -y update --fix-missing
+
+# Install system packages
+RUN apt-get -qqy install \
+    git \
+    vim 
+
+#-------------------------------------------------------------
+#  Install Java OpenJDK 8
+#-------------------------------------------------------------
+RUN \
+  apt-get install -y --fix-missing openjdk-8-jdk
+
+# Define JAVA_HOME variable
+ENV JAVA_HOME /usr/lib/jvm/java-8-openjdk-amd64
+
+#-------------------------------------------------------------
+#  Install Maven
+#-------------------------------------------------------------
+RUN \
+  apt-get install -y maven
+
+#-------------------------------------------------------------
+# Install SPARQL Evaluation Tool
```

Comment:
AIF Validator (or AIF repository)

Reply:
Good catch, I took the base docker from the SPARQL eval tool. I must have missed updating this.